istio-control/istio-autoinject/templates/_helpers.tpl

```diff
@@ -3,7 +3,7 @@
 Expand the name of the chart.
 */}}
 {{- define "sidecar-injector.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- default .Chart.Name .Values.sidecarInjectorWebhook.nameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
This should also go - didn't clean the helpers, but should not be used. Helm leftover

Reply:
https://github.com/istio-ecosystem/istio-installer/issues/66

Will clean them in a follow up pr, too hard to track so many changes in one